docs: add frontmatter to `coin` pages (#23068)
MystenLabs/sui: docs/content/guides/developer/coin/loyalty.mdx

---
title: Loyalty Tokens

> description: The Sui Closed-Loop Token standard can be used to create tokens that are only valid within specific workflows and services. One example of Closed-Loop Tokens is a loyalty token.

Reviewer: ```suggestion
description: Use the Sui Closed-Loop Token standard to create tokens that are only valid within specific workflows and services. One example of Closed-Loop Tokens is a loyalty token.
```

## Example

> The loyalty token example illustrates a loyalty token that is created with the Closed Loop Token standard. If you were to implement this example, the Admin would send `LOYALTY` tokens to the users of your service as a reward for their loyalty. The example creates a `GiftShop` where holders can spend `LOYALTY` tokens to buy `Gift`s.

Reviewer: I think the idea here was that was going to be the name of the example, but I think you're right that it's not spelled out like that anywhere.
